data/coalition/coalition missions.txt

```diff
@@ -1392,3 +1392,160 @@ mission "Arachi Cartoonist 2"
+	passengers 1
+	cargo "unmanned submarines" 60
+	blocked "You need <capacity> to take the next mission. Return here when you have the required space free."
+	to offer
+		has "Coalition: Submarines 2: done"
+	source "Inmost Blue"
+	destination "Dusk Companion"
+	on offer
+		conversation
+			`Apri'ie comes to the spaceport followed by one of his colleagues. "Have to perform one more test, we will, Captain. To assess the subs' repair abilities in hazardous environments, Pale Waters wants, so a quick stop at <destination>, we must make."`
+			choice
+				`	"Alright, we'll head there next then."`
+					goto end
+				`	"Assess their 'repair' abilities? What do you mean?"`
+			`	"Equipped with welders, the arms of the submarines are, so that if greatly damaged deep underwater, one of them is, repaired by the others, it can be," he explains. "Unstable tectonics, <planet> has, so a test repair during one of the nightly tremors, we will conduct. That way, assess their abilities without bothering Second Cerulean's fauna, we can."`
+			label end
+			`	His colleague speaks up. "Headed to <planet>, another team from the research center has. Return to Second Cerulean with the data we have collected, I and the others will. Prepared the subs for your cargo, we already have."`
+			`	You follow him back to your ship, to see the submarines ready on the giant sleds again. After they're brought into your hold, Apri'ie's colleagues say goodbye to him, and he enters your ship once more.`
+				accept
+	on visit
+		dialog `You land on <planet>, but realize that your escort carrying Apri'ie and the unmanned submarines hasn't entered the system yet. Better depart and wait for it.`
+	on complete
+		"coalition jobs" ++
+		payment 130000
+
+mission "Coalition: Submarines 4"
+	landing
+	name "Deliver the Submarines"
+	description "Head to <destination> with Apri'ie and the <cargo>, where the Pale Waters research center will use the new subs to conduct deep-sea expeditions."
+	passengers 1
+	cargo "unmanned submarines" 60
+	to offer
+		has "Coalition: Submarines 3: done"
+	source "Dusk Companion"
+	destination "Second Cerulean"
+	on offer
+		conversation
+			`Apri'ie instructs you to head to hangars in a large port, where a team is waiting for the repair test. He hands you 130,000 credits when you land, and goes to check with the workers unloading the submarines.`
+			`	They are all taken from your cargo and swiftly put in the water, as the team of Kimek boards a large ship that you guess is an old tanker. This ship prepares to follow the subs to the testing area, along with another smaller, newer ship, with a thick crane on the front. Apri'ie approaches you before he boards the tanker, asking, "Come with us for the test, would you like, Captain? Take a while, it might. Not expected until after nightfall, the next tremors are."`
```

Comment:
```suggestion
			`	They are all taken from your cargo and swiftly put in the water, as the team of Kimek boards a large ship that you guess is an old tanker. This ship prepares to follow the subs to the testing area, along with another smaller, newer ship with a thick crane on the front. Apri'ie approaches you before he boards the tanker, asking, "Come with us for the test, would you like, Captain? Take a while, it might. Not expected until after nightfall, the next tremors are."`
```